content/changelogs/README-grml-2025.08/index.md

```diff
@@ -0,0 +1,141 @@
++++
+title = 'Release Notes: Grml 2025.08 - codename Oneinonein'
+
+[params]
+release_version = '2025.08'
+release_packages_date = 2025-08-16
+download_page = 'download'
+kernel_version = '6.12.41'
++++
+
+<a href="/screenshots/"><img align="right" style="margin-left: 20px; border: 0" src="/screenshots/grml_2025.08.jpg" alt="*" /></a>
+
+[Download Now]({{< indirect_ref download_page >}})
+
+### About
+
+Grml is a Debian based live system focusing on the needs of system administrators.
+The {{< param release_version >}} release provides fresh software packages from Debian trixie.
+As usual it also incorporates up to date hardware support and fixes known bugs from previous Grml releases.
+
+About [the codename]({{< ref "faq#releasename" >}}).
+
+### Important changes
+
+* This release is based on the Debian 13 Release named "trixie".
+* We use a [new signing key for the deb.grml.org repositories](https://blog.grml.org/archives/422-Repository-signing-key-transition.html)
+* The grml-live build system incorporated the previously separate packages grml-autoconfig, grml-etc, grml-scripts, grml-udev-config.
+* boot options `lang=` and the `grml-lang` tool have been fixed.
+
+### New and removed features
+
+The highlight of this release is it being based on the Debian 13 trixie release. We spent most of our time on clearning the house. Still some noteworthy changes were implemented:
+
+#### [grml-debootstrap](https://github.com/grml/grml-debootstrap/) - wrapper around debootstrap for installing pure Debian:
+
+* Drop support for Debian 10 codename `buster`
+* Skip udevadm execution on unsupported network interfaces like bonding_masters
+
+#### [grml-live](https://github.com/grml/grml-live/) - build system for creating Grml (based) Linux live systems:
+
+* TBD
+
+#### [grml-scripts](https://github.com/grml/grml-scripts) - console scripts:
+
+* `grml-info`: use links2 without framebuffer driver (dropped upstream)
+* `grml-info`: in X11 always use `x-www-browser`
+* Dropped old scripts: `grml-config-root`, `grml-exec-wrapper`, `grml-setservices`, `isoterm`, `run-multitail`, `noprompt`, `noeject`
+* Moved `run-welcome`, `grml-lang`, `grml-setlang` into grml-live repository
+
+#### [grml-zshrc](/zsh/) - Zsh configuration:
+
+* `zshrc`: increase scrollback to 10000 lines
+* `zshrc`: allow disabling macOS path\_helper
+* `zshrc`: new `cdt` function: cd into a temporary directory
+* `zshrc`: export all known LC\_* variables
+* `zshrc`: removed checkhome hack, only relevant for Live ISOs
+* `zshrc`: removed functions `j`, `iso2utf`, `utf2iso`, `term2iso`, `term2utf`
+* `zshrc`: removed long deprecated xunfunction
+
+### Bits & bolts
+
+* Linux kernel **{{< param kernel_version >}}**
+<!-- misc -->
+* `grml2usb`: new option `--format` replaces `--fat16`, uses FAT32.
+* `grml-hwinfo`: collect ZFS information in JSON format.
+* `grml-tips`: tips are once again numbered starting at 1.
```

Comment:
maybe not worth mentioning here?

nitpick: drop trailing dot (`.`) from end of line, to keep consistent with other listings from above?